app/src/main/java/org/hackillinois/android/view/schedule/ScheduleFragment.java

```diff
@@ -28,6 +29,18 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         viewPager.addOnPageChangeListener(new TabLayout.TabLayoutOnPageChangeListener(tabLayout));
         tabLayout.addOnTabSelectedListener(new TabLayout.ViewPagerOnTabSelectedListener(viewPager));
 
+        ScheduleViewModel scheduleViewModel = new ScheduleViewModel();
+
+        long time = System.currentTimeMillis();
+
+        if (time < scheduleViewModel.getFRIDAY_END()) {
```

Comment:
Don't mix camelCase with UNDERSCORED_UPPERCASE

Reply:
You could make the FRIDAY_END variable a public static final in the file that it's declared in, or a regular private field with a getter method.

Reply:
I think this is why Kotlin doesn't want upper case in variable names. It automatically generates getters and setters depending on var/val, so stuff like this happens.